sidebars.js

```diff
@@ -128,6 +128,15 @@ module.exports = {
             "id": "transport-utp/install"
           }
         ]
+      },
+
+      {
+        "type": "doc",
+        "id": "advanced-topics/networkprofiler-window"
```

Comment:
we're adding network profiler?

Reply:
No, it looks like an error during merging. I'll look into this